x11/xcb/Portfile

```
@@ -2,7 +2,7 @@ PortSystem       1.0
 
 name             xcb
 version          2.4
-revision         3
+revision         4
```

As explained in the MacPorts Guide, don't increase the revision just to fix a build failure that doesn't change installed files.
```suggestion
revision         3
```

---

Ah, I found the documentation to not bump the revision here: https://guide.macports.org/#reference.keywords

Other platforms I do packaging on (typically linux distros), their rule often is if your change modifies the package's source code for any reason, you bump the revision (a/k/a "release" in their terms).  If a build failure is addressed via a change just to the packaging file (`Portfile` here), by say adding a define or changing a compiler or config flag, then leave the revision number alone.

I'll suggest that in the template checklist for PRs, someone could add a line like:
`- [ ] adjusted the revision number based on revision guidelines in [Global Keywords](https://guide.macports.org/#reference.keywords)`
I'd think that's a worthwhile idea even to just be a reminder to have committers think one more time over their change (or not) to the revision number.

Thank you for explaining the bit about commit messages.  I now have enough context to better grok the CommitMessages page.